src/server/server.ts

```diff
@@ -122,6 +135,14 @@ const serve = async ({
     flags.push('--quiet')
   }
 
+  if (inspectSettings && inspectSettings.enabled) {
+    if (inspectSettings.pause) {
+      flags.push(inspectSettings.address ? `--inspect-brk${inspectSettings.address}` : '--inspect-brk')
+    } else {
+      flags.push(inspectSettings.address ? `--inspect${inspectSettings.address}` : '--inspect')
```

Comment:
Shouldn't there be a `=` between the flag and the address? Like `--inspect=1.2.3.4:56`?

Reply:
The address comes with it when passed from CLI e.g `=127.0.01:3030`

Reply:
But why would we expect an address argument starting with a `=`? It feels like CLI should be stripping it out so that Edge Bundler remains generic.

Reply:
Stripped out from CLI side now.